ETL/Python ETL/constants.py

```diff
@@ -3,15 +3,14 @@
 """
 
 CSV_SOURCE_URL = 'https://www.football-data.co.uk/mmz4281/2324/E0.csv'
-## !!! TEMP variable, let's use a local csv for testing purposes
-# CSV_SOURCE_URL = "../../dataset/PremierLeagueTESTDATA.csv"
-
 CURRENT_SEASON_TAG = "2023-2024"
 DEFAULT_MATCHWEEK = 1
+MASTER_CSV_URL = "../../dataset/PremierLeague.csv"
 
 # Database constants
 DB_SERVER = "localhost"
 DB_PORT = "5432"
 DB_USER = "admin"
 DB_PASSWORD = "root"
```

Comment:
just a non-blocking comment

not sure if I forgot to mention this last time, but you can use .env file for env vars like [this](https://dev.to/jakewitcher/using-env-files-for-environment-variables-in-python-applications-55a1)

Reply:
Oh yes! really important to setup a .env file for security purposes as we are going to deploy project to a server in the future.
Thanks for the headsup!